src/normalize.rs

```diff
@@ -77,26 +77,34 @@ const T_COUNT: u32 = 28;
 const N_COUNT: u32 = (V_COUNT * T_COUNT);
 const S_COUNT: u32 = (L_COUNT * N_COUNT);
 
-pub(crate) fn is_hangul(c: char) -> bool {
+const S_END: u32 = S_BASE + S_COUNT - 1;
+const L_END: u32 = L_BASE + L_COUNT - 1;
+const V_END: u32 = V_BASE + V_COUNT - 1;
+const T_END: u32 = T_BASE + T_COUNT - 1;
```

Comment:
For clarity I might call these consts `S_LAST`, etc. and `T_FIRST` below, to make it clear that they're inclusive range ends.